itest/assets_test.go

```diff
@@ -673,6 +683,86 @@ func testAssetBalances(t *harnessTest) {
 	recipients := map[string]uint64{
 		bobAddr.Encoded: bobAddr.Amount,
 	}
+	out, err := wire.NewOutPointFromString(
+		targetAsset.ChainAnchor.AnchorOutpoint,
+	)
+	require.NoError(t.t, err)
+
+	// 1. Fail if Inputs are provided but Outpoint is nil.
+	emptyOutpointPrevId := &wrpc.PrevId{
+		Outpoint:  nil,
+		Id:        assetId,
+		ScriptKey: scriptKey,
+	}
+	_, err = aliceTapd.FundVirtualPsbt(
+		ctxt, &wrpc.FundVirtualPsbtRequest{
+			Template: &wrpc.FundVirtualPsbtRequest_Raw{
+				Raw: &wrpc.TxTemplate{
+					Recipients: recipients,
+					Inputs: []*wrpc.PrevId{
+						emptyOutpointPrevId,
+					},
+				},
+			},
+		},
+	)
+	require.Error(t.t, err)
+	require.Contains(t.t, err.Error(), "index 0 has a nil Outpoint")
```

Comment:
nit: can combine this and `require.Error()` into a single call to `require.ErrorContains()`.